public/locales/es/translations.json

```
@@ -0,0 +1,49 @@
+{
+	"root.description": "Una alternativa al ecosistema de Nix",
+
+	"header.community": "Comunidad",
+
+	"goals.title": "Objetivos",
+	"goals.independent.title": "Independiente",
+	"goals.independent": "Mantendremos una bifurcación ("fork") de Nix, NixPkgs y NixOS. Se considerará soporte adicional para proyectos como Nix Darwin y Home Manager.",
```

```suggestion
	"goals.independent": "Mantendremos una bifurcación (fork) de Nix, NixPkgs y NixOS. Se considerará soporte adicional para proyectos como Nix Darwin y Home Manager.",
```

Sorry, I messed up here! The double quotes is making the string end earlier.

---

Dw, already pulled the change. :)